src/shared/pretty-print.h

```diff
@@ -4,6 +4,10 @@
 #include "glyph-util.h"
 #include "terminal-util.h"
 
+#define CYLON_BUFFER_EXTRA (2*STRLEN(ANSI_RED) + STRLEN(ANSI_HIGHLIGHT_RED) + 2*STRLEN(ANSI_NORMAL))
+
+void draw_cylon(char buffer[], size_t buflen, unsigned width, unsigned pos);
```

Comment:
maybe change first arg to `char buffer[static CYLON_BUFFER_EXTRA]`

Reply:
No, the buffer size must be larger than `CYLON_BUFFER_EXTRA`.